src/target/riscv/riscv-013.c

```diff
@@ -686,6 +715,25 @@ static int dmi_write_exec(struct target *target, uint32_t address,
 	return dmi_op(target, NULL, NULL, DMI_OP_WRITE, address, value, true, ensure_success);
 }
 
+static bool dbgbase_verify(struct target *target)
```

Comment:
If you like, you can give this function a more descriptive name:

```suggestion
static bool check_dbgbase_exists(struct target *target)
```